src/main/resources/Holidays_gr.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@nikoszagk can you please format this file? The format is defined by the .editorconfig

Reply:
I inserted a black line at the end

Reply:
That will not help to successfully build the uv. Please format the file correctly :)

Reply:
I edited correctly the .xml file :)

Reply:
Can you run `mvn editorconfig:format` ? :)

Reply:
Done :) :)